backend/azure-functions/src/Services/Auth/facebook-auth.ts

```diff
@@ -42,7 +43,11 @@ export async function authenticate(
 
     let userId: string | null = userTokenCache.getUserIdFromValidToken(userToken);
     if (userId != null) {
-        context.log.info('Authorization token found on cache');
+        if (logUserId) {
```

Comment:
Why would you sometimes not want to log the userId?

Reply:
Any information that we save on our users is something that we need to treat carefully. So I prefer not to save what we don't need.
In the admin app, I would like to save the users, to audit who performed operations - and anyway it should be only us. But in the customer app I prefer to not log if we don't need.